std/node/README.md

```diff
@@ -0,0 +1,7 @@
+# Deno Node compatibility
+
+This module is meant to have a compatibility layer for the
+[nodeJS standard library](https://nodejs.org/docs/latest-v11.x/api/).
```

Comment:
this should be TLS version

Reply:
*LTS

Reply:
There is no link directly pointing to LTS version, updated it to aim v12 as it's LTS: https://nodejs.org/en/